src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperOutput.cs

```diff
@@ -36,9 +34,9 @@ public TagHelperOutput(string tagName, [NotNull] IDictionary<string, string> att
         {
             TagName = tagName;
             Attributes = new Dictionary<string, string>(attributes, StringComparer.OrdinalIgnoreCase);
-            PreContent = string.Empty;
-            _content = string.Empty;
-            PostContent = string.Empty;
+            PreContent = new DefaultTagHelperContent();
```

Comment:
Hmmm, @dougbu / @rynowak what do you think about creating these things only if they're requested? Not saying I want to pollute with `Lazy<DefaultTagHelperContent>` but even just a null check in the getter.